src/pages/Diet/components/AddImage.jsx

```diff
@@ -0,0 +1,53 @@
+import axios from 'axios';
+import { useState } from 'react';
+
+export function AddImage() {
+  let [link, setLink] = useState('');
+  let [files, setFiles] = useState(null);
```

Comment:
if we want to change state in React, we'll use setFunction, we won't assign new value directly.
use const replace let to make declaration.

Reply:
What's `setFunction` ?
The condition here I thought was set an empty string to `link` and set a `null` to `files` for the default value.